datacite/schema42.py

```diff
@@ -0,0 +1,431 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of DataCite.
+#
+# Copyright (C) 2016 CERN.
```

Comment:
2019

Reply:
I added Caltech and left in CERN (alot is the same as schema41)